File: library/src/main/java/com/mikepenz/aboutlibraries/util/Colors.java
package com.mikepenz.aboutlibraries.util;

import android.os.Parcel;

import java.io.Serializable;

/**
 * Created by Yoav.
 */
public class Colors implements Serializable {
    public int appBarColor;
    public int statusBarColor;

    public Colors(int toolbarColor, int statusBarColor) {
        this.appBarColor = toolbarColor;
        this.statusBarColor = statusBarColor;
    }

    private Colors(Parcel in) {
        this.appBarColor = in.readInt();
        this.statusBarColor = in.readInt();
    }
}

File: library/src/main/java/com/mikepenz/aboutlibraries/util/Colors.kt
package com.mikepenz.aboutlibraries.util

import android.os.Parcel
import java.io.Serializable


/**
 * Created by Mike Penz.
 */
class Colors : Serializable {
    var appBarColor: Int = 0
    var statusBarColor: Int = 0

    constructor(toolbarColor: Int, statusBarColor: Int) {
        this.appBarColor = toolbarColor
        this.statusBarColor = statusBarColor
    }

    private constructor(input: Parcel) {
        this.appBarColor = input.readInt()
        this.statusBarColor = input.readInt()
    }
}
